frontend/src/components/ui/AppDocumentUpload.vue

```diff
@@ -64,6 +92,23 @@ export default {
       type: Array,
       default: () => [],
     },
+    header: {
```

Comment:
Is this prop required? It always seems to be paired with documentType and despite being a String is used as a Boolean. I think documentType is enough.

Reply:
It is required... if populated it displays custom header, if not displays previous functionality of no header and just the doc validation user info message.

Reply:
Removed header prop.